code/game/machinery/newscaster.dm

```diff
@@ -1112,10 +1112,10 @@ GLOBAL_LIST_EMPTY(allCasters)
 		return has_read[reader.ckey]
 
 	if(admin_msg)
-		var/positive = admin_mood_amount >= 0 ? TRUE : FALSE
-		SEND_SIGNAL(reader, COMSIG_ADD_MOOD_EVENT, "horoscope_admin", positive ? /datum/mood_event/horo_good : /datum/mood_event/horo_bad)
-		message_admins("[reader] has read the admin horoscope.")
-		return "<span class='[positive ? "nicegreen" : "warning"]'>[admin_msg]</span>"
```

Comment:
don't remove the span

Reply:
this is the actual horoscope written to chat, it would make no sense to span it because the newspaper already spans it with 'notice'

Reply:
Okay